plugins/MauticCitrixBundle/EventListener/FormSubscriber.php

```diff
@@ -600,4 +599,18 @@ public function onFormBuilder(Events\FormBuilderEvent $event)
             }
         }
     }
+
+    /**
+     * @param Collection<int,Field> $fields
+     */
+    protected function getEmailFieldAlias(Collection $fields): string
```

Comment:
Any reason for protected method?
```suggestion
    private function getEmailFieldAlias(Collection $fields): string
```